docs/static/resources/openapi.json

```diff
@@ -17912,7 +17912,7 @@
                       "description": "Menu items in a forest like data structure",
                       "items": {
                         "properties": {
-                          "childs": {
+                          "children": {
```

Comment:
Eek! Houston, we have a problem. @dpgaspar can correct me here, but I believe the underlying library (Flask Appbuilder) expects "childs" here, so this is correct implementation, by way of a typo in the base library. We might want to unwind these. 

But again, this is passing CI... so... I'm not sure what's up with that.